policyengine_us/variables/gov/states/az/tax/income/credits/az_increased_excise_tax_credit.py

```diff
@@ -0,0 +1,25 @@
+from policyengine_us.model_api import *
+
+
+class az_increased_excise_tax_credit(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arizona Increased Excise Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.AZ
+    reference = "https://www.azleg.gov/viewdocument/?docName=https://www.azleg.gov/ars/43/01072-01.htm"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.az.tax.income.credits.increased_excise
+        agi = tax_unit("adjusted_gross_income", period)
+        filing_status = tax_unit("filing_status", period)
+        max_income = p.income_threshold[filing_status]
+        eligible = agi <= max_income
```

Comment:
Could you make a separate variable `az_increased_excise_tax_credit_eligible` and use that in the `defined_for` here? It'll speed up computations and modularize more.